Rubberduck.Core/UI/CodeExplorer/CodeExplorerControl.xaml

```diff
@@ -583,9 +589,9 @@
                 <Separator />
 
                 <Button Command="{Binding CopyResultsCommand}" CommandParameter="{Binding ElementName=ProjectTree}">
-                    <Image Source="pack://application:,,,/Rubberduck.Resources;component/Icons/Fugue/document-copy.png">
```

Comment:
kudos for cleaning up the pack URIs here! 👍

Reply:
Those were already merged with a previous pull request https://github.com/rubberduck-vba/Rubberduck/commit/a830af6fa45d0b1997439ca28965e6d0a5bb9382. I inadvertently polluted this branch when I created it without realizing.